devices/generic/items/state_preset_effect_item.json

```diff
@@ -0,0 +1,8 @@
+{
+  "schema": "resourceitem1.schema.json",
+  "id": "state/preset_effect",
```

Comment:
Please re-use `state/effect` and `cap/color/effects`.  Also, "preset" sounds like a `config` item, not a `state` item.